qiskit/utils/backend_utils.py

```diff
@@ -37,7 +37,7 @@ def __init__(self) -> None:
 def _get_backend_interface_version(backend):
     """Get the backend version int."""
     backend_interface_version = getattr(backend, "version", None)
-    # Handle deprecated BaseBackend based backends which have a version()
+    # Handle deprecated Backend based backends which have a version()
     # method
     if not isinstance(backend_interface_version, int):
         backend_interface_version = 0
```

Comment:
I think these lines are redundant now?

Reply:
Yeah they are, it should never be triggered now, I'll drop them too

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7886/commits/090ae177611bcfbb1628113ad1cc0c657463977d